src/cli.rs

```diff
@@ -27,25 +29,36 @@ pub fn bootstrap_cli() {
     let path_to_local_release_binary =
         local_binary_project_directory.join(format!("target/release/{LOCAL_BINARY_PROJECT_NAME}"));
     let command_line_args = env::args_os().collect::<Vec<_>>();
+
+    let span = debug_span!("parse args").entered();
+
     let args = Args::parse_from(command_line_args.iter().cloned());
+
+    span.exit();
+
     if should_regenerate_local_binary(&config_file_path, &path_to_local_release_binary, &args) {
         regenerate_local_binary(&local_binary_project_directory, &Path::new("..").join(".."));
     }
-    let mut handle = Command::new(path_to_local_release_binary)
-        .args(command_line_args.into_iter().skip(1))
-        .spawn()
-        .unwrap();
+    let mut command = Command::new(path_to_local_release_binary);
+    command.args(command_line_args.into_iter().skip(1));
+    if let Ok(rust_log) = env::var("RUST_LOG") {
+        command.env("RUST_LOG", rust_log);
```

Comment:
Maybe this should be a more generalized "forwarding" of environment variables to the custom local binary execution?

Reply:
Yeah, I think that probably makes sense.